### DISCUSSION TITLE
How do I use tomcat in code-serve

### DISCUSSION THREAD

**Discussion by Lailah123** [coder/code-server#5653] [Q&A]
There are no supported plug-ins

**Comment by jsjoeio** [comment]
What is tomcat and what are you trying to do? That'll help us help you faster!

**Reply by lg-nineoneninethreenine** [reply]
It's Apache Tomcat. Maybe they are asking us how to access it?

**Comment by Lailah123** [comment]
The tomcat plug-in is like the Tomcat for Java plug-in or Community Server Connectors in vscode
<img width="1455" alt="image" src="https://user-images.githubusercontent.com/30382615/196364934-d2db3358-21ca-48b8-9cb7-d0cb7b542105.png">

**Reply by jsjoeio** [reply]
Ah okay. And so you're saying it's not supported in code-server? Is it because of the VS Code version?
